docs/release-notes.md

```diff
@@ -3,6 +3,34 @@
 - Table of Contents
 {:toc}
 
+## Version 1.2.0
+
+The 1.2.0 release of MCUboot brings a lot of fixes/updates, where much of the
+changes were on the boot serial functionality and imgtool utility. There are
+no breaking changes in MCUBoot functionality, but some of the CLI parameters
+in imgtool were changed (either removed or added or updated).
+
+### About this release
+
+- imgtool accepts .hex formatted input
+- Logging system is now configurable
+- Zephyr build system switched to Kconfig; accepts .pem files in build system
+  to autogenerate required key arrays used internally
+- imgtool updates:
+  + "create" subcommand can be used as an alias for "sign"
+  + To allow imgtool to always perform the check that firmware does not
+    overflow the status area, `--slot-size` was added and `--pad` was updated
+    to act as a flag parameter.
+  + `--overwrite-only` can be passed if not using swap upgrades
+  + `--max-sectors` can be used to adjust the maximum amount of sectors that
+    a swap can handle; this value must also be configured for the bootloader
+  + `--pad-header` substitutes `--included-header` with reverted semantics,
+    so it's not required for firmware built by Zephyr build system
+
```

Comment:
zephyr footprint was reduced and  source code simplified thanks to:
- zephyr build switched to using flash_map and tinycborr modules build in zephyr-project.
- zephyr serial recovery build doesn't require build with newlib-c anymore

Reply:
Added this with small changes, thanks!